app/logic/Contacts/VCard/VCard.ts

```diff
@@ -196,7 +196,7 @@ function makeContactEntry(entry: ICalEntry, protocol?: string, isURI = true, str
     value,
     /work/i.test(entry.properties.type) ? "work" : /home/i.test(entry.properties.type) ? "home" : "other",
     protocol,
-    sanitize.integer(entry.properties.pref, ContactEntry.defaultPreference));
+    sanitize.integer(entry.properties.pref, /pref/i.test(entry.properties.type) ? 1 : ContactEntry.defaultPreference));
```

Comment:
Q: In modern browser javascript/typescript, what's the fastest way to compare 2 strings case-insensitive? Which is the most readable way? Which way has the best balance between runtime speed and code readability?
A: https://www.perplexity.ai/search/what-s-the-fastest-way-to-comp-UzeDgN5aQV673aL7kq1.OA

toLowerCase() / toUpperCase()	Fastest	Most clear
RegExp with /i flag	Slowest	Less clear